lib/api/apiUtils/object/applyZenkoUserMD.js

```diff
@@ -0,0 +1,19 @@
+const { zenkoIDHeader } = require('arsenal').constants;
+
+const _config = require('../../../Config').config;
+
+/**
+ * applyZenkoUserMD - if request is within a Zenko deployment, apply user
+ * metadata called "zenko-source" to the object
+ * @param {Object} metaHeaders - user metadata object
+ * @return {undefined}
+ */
+function applyZenkoUserMD(metaHeaders) {
+    if (process.env.REMOTE_MANAGEMENT_DISABLE === '0' &&
```

Comment:
by default, isn't `REMOTE_MANAGEMENT_DISABLE` undefined and unspecified? Maybe it'd be better to check if someone explicitly set `REMOTE_MANAGEMENT_DISABLE=1` and work from there

Reply:
I think `REMOTE_MANAGEMENT_DISABLE` will only be equal to '0' when orbit is enabled. You're right about this being undefined for cases like running file or mem backend, but we can avoid setting this user md field in those cases